autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/imputation/SimpleImputer.py

```diff
@@ -29,45 +29,36 @@ def __init__(
         categorical_strategy: str = 'most_frequent'
     ):
         """
-        Parameters
-        ----------
-        random_state: Optional[Union[np.random.RandomState, int]] = None
-            The random state to use for the imputer
-
-        numerical_strategy: str = 'mean',
-            The strategy to use for imputing numerical columns.
-            Can be one of ['mean', 'median', 'most_frequent', 'constant', 'constant_!missing!']
-
-            Note:
-                Using 'constant' defaults to fill_value of 0 where 'constant_!missing!'
-                uses a fill_value of -1. This behaviour should probably be fixed.
-
-        categorical_strategy: str = 'most_frequent'
-            The strategy to use for imputing categorical columns.
-            Can be one of ['mean', 'median', 'most_frequent', 'constant_zero']
+        Note:
+            Using 'constant' defaults to fill_value of 0 where 'constant_!missing!'
+            uses a fill_value of -1. This behaviour should probably be fixed.
+
+        Args:
+            random_state (Optional[Union[np.random.RandomState, int]]):
+                The random state to use for the imputer.
+            numerical_strategy (str: default='mean'):
+                The strategy to use for imputing numerical columns.
+                Can be one of ['mean', 'median', 'most_frequent', 'constant', 'constant_!missing!']
+            categorical_strategy (str: default='most_frequent')
+                The strategy to use for imputing categorical columns.
+                Can be one of ['mean', 'median', 'most_frequent', 'constant_zero']
         """
         super().__init__()
         self.random_state = random_state
         self.numerical_strategy = numerical_strategy
         self.categorical_strategy = categorical_strategy
 
     def fit(self, X: Dict[str, Any], y: Optional[Any] = None) -> BaseImputer:
-        """
-        The fit function calls the fit function of the underlying model
-        and returns the transformed array.
+        """ Fits the underlying model and returns the transformed array.
 
-        Parameters
-        ----------
-        X: np.ndarray
-            The input features to fit on
+        Args:
+            X (np.ndarray):
+                The input features to fit on
+            y (Optional[np.ndarray]):
+                The labels for the input features `X`
 
-        y: Optional[np.ndarray]
-            The labels for the input features `X`
-
-        Returns
-        -------
-        SimpleImputer
-            returns self
+        Returns:
+            SimpleImputer: returns the object itself
```

Comment:
Can you add a new line after `SimpleImputer:`? Otherwise, the formatting of documentation generated by Sphinx causes issues.